cdap-distributions/src/packer/files/cdap-sdk.json

```diff
@@ -1,6 +1,6 @@
 {
   "cdap": {
-    "version": "4.1.0-1"
+    "version": "4.1.1-SNAPSHOT"
```

Comment:
Ok to just bump it to 4.1.1? Otherwise, we may not find this later, when we're removing snapshot?